src/main/java9/module-info.java

```diff
@@ -4,8 +4,4 @@
     requires static java.validation;
 
     exports org.openapitools.jackson.nullable;
-
-    provides com.fasterxml.jackson.databind.Module with org.openapitools.jackson.nullable.JsonNullableModule;
-    provides javax.validation.valueextraction.ValueExtractor with org.openapitools.jackson.nullable.JsonNullableValueExtractor;
-    provides jakarta.validation.valueextraction.ValueExtractor with org.openapitools.jackson.nullable.JsonNullableJakartaValueExtractor;
```

Comment:
Maybe leave this in, but commented out that it can be re-added when either of the below are true.
1. Baseline JDK is 25 (to avoid telling users that they can build with 25+, and still use something lower).
2. Javax is dropped, so that there isn't a need for anything optional.

Just a suggestion, but not a problem either way.